### ISSUE TITLE
Can I add a / to the end of post url for cloudflare pages

### ISSUE THREAD

**Issue by rhuddlestone** [onwidget/astrowind#55]
Hi

I have deployed a site to Cloudflare pages.  When I visit a URL, https://mysite.com/post-name Cloudflare redirects it to https://mysite.com/post-name/ (with the trailing /) as a 308 redirect.  This doesn't happen in the Dev environment and appears to be a known "feature" in Cloudflare pages.  This is not ideal as it adds an extra request, so some load time.

Is there any way I could add the trailing / to generated links via the theme (e.g., in my blog post list, have the URLs be https//mysite.com/my-post/ instead of https//mysite.com/my-post)?

Thanks

Rob

**Comment by rhuddlestone** [comment]
Sorry, I should have just found the below doc.


https://docs.astro.build/en/reference/configuration-reference/#trailingslash

This should fix my problem.

**Comment by rhuddlestone** [comment]
Hi 

I took a look in the permalinks.js file and made a change. 

const createPath = (...params) => '/' + params.filter((el) => !!el).join('/') + '/';  (Added the + '/') at the end of the function.

Is this a reasonable way of doing this or do you think I am creating more problems with this?

Thanks

Rob

**Comment by rhuddlestone** [comment]
Hi @prototypa 

That fixed it, there is just one more place, and that is the pagination on the archives. The "Older posts" and "Newer Posts" links do not have the trailing slash.

I really appreciate your quick responses.  I will send you a few dollars via your buy me a coffee link,

Great work

**Comment by rhuddlestone** [comment]
Hi @prototypa 

Thanks for the kind words.

Everything is working great now.  I am using astrowind for an info blog on WordPress, and I moved it due to the cost os hosting on WordPress and all the bloat. 

My load times have gone from >3secs (with expensive hosting) to <400msecs (with free hosting on CloudFlare pages).

I have found these issues via Google Search Console and Ahrefs crawl reports.

As for new features, the two main things are already on your roadmap.

Create component or utilities for latest posts
Create component or utilities for related posts

I also see a merge for social share buttons that will be very nice.

Keep up the great work.
